composer.json

```
@@ -115,7 +115,8 @@
         ],
         "post-autoload-dump": [
             "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
-            "@php artisan package:discover"
+            "@php artisan package:discover",
+            "@php artisan passport:keys"
```

I've made a special command to handle this.
This way if the files are already here, or if `PASSPORT_PRIVATE_KEY` and `PASSPORT_PUBLIC_KEY` env variable are present, the key will not be overwritten.

```suggestion
            "@php artisan monica:passport"
```

---

Okay I'll try this!